src/layers/core/path-layer/path-layer-vertex-64.glsl.js

```diff
@@ -163,6 +167,14 @@ vec3 lineJoin(vec2 prevPoint64[2], vec2 currPoint64[2], vec2 nextPoint64[2]) {
 
   vCornerOffset = offsetVec * offsetDirection * offsetScale;
 
+  // Generate variables for dash calculation
+  vDashArray = instanceDashArrays;
+  vPathLength = L / width;
```

Comment:
I am not a fan of abbreviated identifiers (of which there are many in this shader), single letter ones even less so.